plasmapy/particles/__init__.py

```diff
@@ -5,6 +5,7 @@
 # __all__ will be auto populated below
 __all__ = []
 
+from plasmapy.particles._special_particles import particle_zoo
```

Comment:
The `particle_zoo` documentation is a bit odd, since it inherits the docstring from `ParticleZoo`.  Basically it's a variable with a class like documentation, minus the the documentation for the methods.  I would consider exposing `ParticleZoo` to the `plasmapy.particles` namespace and defining `particle_zoo` in `plasmapy.particles` with an appropriate new docstring.  I say define `particle_zoo` in `plasmapy.particles` because if you define it in `_special_particles` the docstring won't (at least in my experience) carry over to the instance in `plasmapy.particles`.

I'd see this looking something like the following in `plasampy/particles/__init__.py`

```python
from plasmapy.particles._special_particles import ParticleZoo

particle_zoo = ParticleZoo()
"""
Data container for representations of "special" particles, like leptons, baryons, etc.

An instance of `~plasmapy.particles._special_particles.ParticleZoo`.
"""
```

I do think for completeness, if `particle_zoo` is in our documentation then it must be accompanied with `ParticleZoo`.

---

I would also consider having `ParticleZoo` inherit from `collections.UserDict` so `particle_zoo` would become the `taxonomy_dict`.

Reply:
Since `particle_zoo` is an instance of a class and thus it has a `__doc__` attribute (unlike most variables), think we can still define it in `_special_particles`.  I think it would then have to look like...

**In `__init__.py`**

```python
from plasmapy.particles._special_particles import particle_zoo, ParticleZoo
```

**In `_special_particles.py`**

```python
particle_zoo = ParticleZoo()
particle_zoo.__doc__ = """
Data container for representations of "special" particles, like leptons, baryons, etc.

An instance of `~plasmapy.particles._special_particles.ParticleZoo`.
"""
```